Library/Util.ts

```diff
@@ -140,10 +140,11 @@ class Util {
                 var property = obj[field];
                 var propertyType = typeof property;
                 if (propertyType !== "string") {
-                    if (property && typeof property.toString === "function") {
+                    if (property != null && typeof property.toString === "function") {
```

Comment:
`p != null` is the same as `p !== null && p !== undefined`. I wanted to make sure that it's safe to call  `toString`.